.github/workflows/ci.yml

```diff
@@ -22,12 +26,16 @@ jobs:
         run: make check
       - name: distcheck
         run: make distcheck
-  test-ubuntu-oldest:
-    runs-on: ubuntu-16.04
+
+  test-centos:
```

Comment:
Can we do a matrix of Docker images like this:

```yaml
  test-red-hat:
    strategy:
      matrix:
        image:
          - centos:latest
          - centos:6
          - fedora:latest
          - fedora:26
```

Reply:
Nope, cause they use different commands.